.github/.env

```diff
@@ -8,3 +8,4 @@ KEY_VAULT=kv-sswwebsite-9eb3
 APP_INSIGHT_PROD=appInsight-sswwebsite-9eb3
 APP_INSIGHT_DEV=appInsight-sswwebsite-dev-9eb3
 NEXT_PUBLIC_CHATBASE_BOT_ID=gnbZXuG_5tgI_R3yzKcOn
+TARGET_URL="https://app-sswwebsite-9eb3-pr-1464.azurewebsites.net/consulting/angular"
```

Comment:
I think the URL host should be in the weekly workflow i.e. https://ssw.com.au
The test should have `/consulting/angular`

Reply:
done - I replaced it with HOST_URL and added the route in the test itself